securedrop/bin/run-test

```diff
@@ -5,7 +5,7 @@ set -euo pipefail
 
 source "${BASH_SOURCE%/*}/dev-deps"
 
-run_xvfb &
+#run_xvfb &
```

Comment:
Hm.. why is this commented out? I didn't see anything in the commit messages about this, apologies if I missed it

Reply:
Ah, this is my mistake. I will clean things out in a commit based on the review.

Reply:
Looks like the the removal of `run_xvfb` has not yet been addressed; I will attempt patching locally and fix if tests still pass.

Reply:
Can confirm that re-enabling `run_xvfb` locally does not break functional tests, so will include that modification in the upcoming cleanup.